File: array/2738.js
const fs = require('fs');
// const input = fs.readFileSync(__dirname + '/2738').toString().trim().split('\n');
let input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

const met = input.shift().split(" ").map(v => Number(v));

const a = [];
const b = [];

input.forEach((v, i) => i < met[0] ? a.push(v.split(" ").map(v => Number(v))) : b.push(v.split(" ").map(v => Number(v))))

a.map((e, i) => e.reduce((pre, cur, idx) => {
    pre.push(cur + b[i][idx])
    return pre;
  }, [])).forEach(v => console.log(v.join(" ")))

File: array/2738.py
# def add_one(n):
#     return n + 1

f = list(map(int, input().split()))

a = []
b = []
result = []

for i in range(f[0] * 2):
    if i < f[0]:
        a.append(list(map(int, input().split())))
    else:
        b.append(list(map(int, input().split())))


for idx, i in enumerate(a):
    temp = []
    for idxx, j in enumerate(i):
        temp.append(j + b[idx][idxx])

    result.append(temp)

for r in result:
    print(' '.join(str(e) for e in r))